base/src/com/google/idea/blaze/base/sync/ProjectUpdateSyncTask.java

```diff
@@ -346,6 +347,7 @@ private boolean updateProject(
             throw e;
           } catch (Throwable e) {
             IssueOutput.error("Internal error. Error: " + e).submit(context);
+            context.output(new PrintOutput(e.getMessage(), PrintOutput.OutputType.ERROR));
```

Comment:
Shouldn't this be reported by the issue output above?

Reply:
Nvm, ToolWindowScope has no sink for issues.

Reply:
@mai93 Maybe we can add a sink for issues to the ToolWindowScope with `Propagation.Continue`?